source/adminguide/templates.rst

```diff
@@ -383,6 +385,156 @@ When templates are deleted, the VMs instantiated from them will continue
 to run. However, new VMs cannot be created based on the deleted
 template.
 
+Working with ISOs
+===================
+
```

Comment:
is this addition relevant to this pr or did it creep in?

Reply:
i don't remember, i suspect that i found it in 'the wrong place' when structuring the additional documentation.